src/sentry/constants.py

```diff
@@ -167,7 +167,7 @@ def get_all_languages():
     'project', 'errors', 'event_id', 'message', 'checksum', 'culprit', 'fingerprint', 'level',
     'time_spent', 'logger', 'server_name', 'site', 'received', 'timestamp', 'extra', 'modules',
     'tags', 'platform', 'release', 'dist', 'environment', 'transaction', 'key_id', '_meta',
-    'applecrashreport', 'device', 'repos', 'query', 'type', 'hashes',
```

Comment:
This is such that `repos` is discarded as unknown attribute.

Reply:
Updated this part to remove legacy interfaces without an error message (see `CLIENT_IGNORED_ATTRS`).